src/rules/no-eol-whitespace/README.md

```diff
@@ -39,3 +39,30 @@ a { color: pink; }
 /* something
  * something else */
 ```
+
+## Optional options
+
+### `ignore: ["empty-lines"]`
+
+#### `"empty-lines"`
+
+Don't disallow end-of-line whitespace for lines that are only whitespace, "empty" lines.
```

Comment:
I think "Allow end-of-line whitespace for lines that are only whitespace, "empty" lines." is more consistent with these descriptions [else where](https://github.com/stylelint/stylelint/tree/master/src/rules/selector-no-type#descendant).